src/SWP/Bundle/ContentBundle/Loader/ArticleMediaLoader.php

```diff
@@ -92,7 +92,7 @@ public function load($type, $parameters = [], $responseType = LoaderInterface::C
             }
 
             $criteria = $this->applyPaginationToCriteria($criteria, $parameters);
-            if ($criteria->get('article')->getMedia()->isInitialized()) {
```

Comment:
Why you removed it? there is no point to lazy load it here - in else we have faster method for count.